### ISSUE TITLE
Live demo

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-tiles#2]
I can use `datasette-basemap` and my San Francisco demo database.

**Comment by simonw** [comment]
```
datasette publish cloudrun San_Francisco.mbtiles \
    --service=datasette-tiles-demo \
    --about=datasette-tiles \
    --about_url=https://github.com/simonw/datasette-tiles \
    --install datasette-basemap \
    --install datasette-tiles
```

**Comment by simonw** [comment]
https://datasette-tiles-demo-j7hipcg4aq-uc.a.run.app/-/tiles/basemap - should go live at https://datasette-tiles-demo.datasette.io/ as soon as the certificate is issued (usually takes about 15 minutes).

**Comment by simonw** [comment]
I'll close this after I publish a successful release with it.

**Comment by simonw** [comment]
https://github.com/simonw/datasette-tiles/runs/1818690016?check_suite_focus=true
```
ERROR: Could not find a version that satisfies the requirement datasette-tiles==0.3
ERROR: No matching distribution found for datasette-tiles==0.3
```
I'm going to increase the sleep delay.

**Comment by simonw** [comment]
That worked! The GitHub Actions `publish` workflow successfully deployed the latest demo when I released 0.4.
